OWNERS

```diff
@@ -1,10 +1,13 @@
 approvers:
 - bparees
+- awgreene
+- perdasilva
 - joelanford
 - perdasilva
```

Comment:
i don't think we need Per twice :)

maybe alpha-sort the list to help avoid that problem in the future.

Reply:
(though if we had two of Per, that would certainly be awesome)

Reply:
Ha! Yes it would!